doc/lua_api.txt

```diff
@@ -1777,6 +1777,13 @@ supported. It is a string, with a somewhat strange format.
 Spaces and newlines can be inserted between the blocks, as is used in the
 examples.
 
+Position and size units are inventory slots, `X` and `Y` position the formspec
```

Comment:
I'm not sure if it is clear what X refers to because X appears later in the formspec element descriptions.

Reply:
In some way it's a prelude to the documentation which follows afterwards. I think that's fine.